CRM/Civioffice/DocumentRendererType/LocalUnoconv.php

```diff
@@ -581,13 +582,16 @@ protected function runCommand($command)
     /**
      * wait for the unoconv resource to become available
      */
-    protected function lock()
+    protected function lock(): void
     {
-        $lock_file_path = $this->unoconv_lock_file_path;
-        if ($lock_file_path) {
-            $this->unoconv_lock_file = fopen($lock_file_path, "w+");
+        if ($this->unoconv_lock_file_path) {
```

Comment:
The property is not a bool and should either be a string or not set, let's use `isset()` here?

Reply:
I added the PHP type hint `?string` to the property and changed the condition to `NULL !== $this->unoconv_lock_file_path && '' !== $this->unoconv_lock_file_path`.